tests/rptest/tests/cloud_storage_timing_stress_test.py

```diff
@@ -515,6 +515,8 @@ def test_cloud_storage(self, cleanup_policy):
             r"Error in hydraton loop: .*Connection reset by peer",
             r"failed to hydrate chunk.*Connection reset by peer",
             r"failed to hydrate chunk.*NotFound",
+            # This error is only acceptable if the parser is at end of stream.
+            r"segment_reader is stuck.*parser error state: parser_errc::end_of_stream",
```

Comment:
I think that with the change in remote_segment this could be removed

Reply:
You're right, I missed pushing the change. Fixed now